README.md

```diff
@@ -311,6 +315,34 @@ MFA to the right AWS account using the following command
 
     tx-mfa
 
+## Production Deployment
+
+### Prerequisites
+
+The production pipeline terraform stack must be deployed
+
+    make deploy-deployment-pipelines PROFILE=tools ENVIRONMENT=dev
+
+### How to deploy
```

Comment:
Might be worth adding something in this section to explain what happens when you run the tag-commit command, e.g. explaining the tag goes to github and the corresponding pipeline picks it up and deploys it.